.github/workflows/ci.yml

```diff
@@ -237,6 +238,23 @@ jobs:
       binary: generate_isograph_fixtures
       params: --dir ./crates/isograph_lang_parser/fixtures/
 
+  build-swc:
+    name: Build swc
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: stable
+          components: rustfmt
+      - name: Install wasm target
+        shell: bash
+        run: |
+          rustup target add wasm32-wasip1
```

Comment:
https://github.com/actions-rust-lang/setup-rust-toolchain?tab=readme-ov-file#inputs I think you can just do `target: wasm32-wasip1`